client/src/App/components/groups/createEdit/GroupCreateEditSurface.jsx

```diff
@@ -36,6 +36,9 @@ class GroupCreateEditSurface extends React.Component {
         // topics get passed from the form as a single string value, 
         // but must be passed to the server as an array
         group.topics = [group.topics];
+        if (!group.leaders) {
+            group.leaders = [];
```

Comment:
They do default to that user, but it happens in the server code. Lines 60-62 in groupsController. Figured since the logic for that was there, we need only default it to an empty array here.